Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -1210,6 +1213,16 @@ public IEnumerable<TItem> Data
     /// </summary>
     [Parameter] public EventCallback<AutocompleteReadDataEventArgs> ReadData { get; set; }
 
+    /// <summary>
+    /// Event handler used to detect when the autocomplete is closed.
+    /// </summary>
+    [Parameter] public EventCallback<AutocompleteCloseEventArgs> Closed { get; set; }
```

Comment:
Should we add a prefix to the parameter name, eg, `MenuCloses` so that it is clearer what is actually being closed? Or is `Closed` good enough? @David-Moreira

Reply:
I feel like `Opened` and `Closed` are fine. It's easy to infer what they relate to in my opinion.

Reply:
@stsrki would you agree?

Reply:
Yeah, seems fine.